index.html

```diff
@@ -566,5 +566,121 @@ <h1 class="primary-text">About me</h1>
         </div>
       </div>
     </section>
+
+    <section class="form-section">
+      <p class="text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/mgeqkqgz" method="post">
+        <input
+          type="text"
+          name="name"
+          placeholder="Full Name"
+          required
+          maxlength="30"
+        />
+        <input type="email" name="email" placeholder="Email Address" required />
+        <textarea maxlength="500" name="message" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Please note that All input fields should have the `required` attribute to activate the validation on submission. the textarea (the message input) doesn't have a `required` attribute kindly add that 🙏